UnityProject/Assets/Scripts/Player/PlayerMove.cs

```diff
@@ -118,9 +118,10 @@ public bool IsSwappable
 
 	[HideInInspector] public PlayerNetworkActions pna;
 
-	[FormerlySerializedAs("speed")] public float RunSpeed = 6;
-	public float WalkSpeed = 3;
-	public float CrawlSpeed = 0.8f;
+	[FormerlySerializedAs("speed")] public float InitialRunSpeed = 6;
+	[HideInInspector][SyncVar] public float RunSpeed = 6;
+	[SyncVar] public float WalkSpeed = 3;
+	[SyncVar] public float CrawlSpeed = 0.8f;
```

Comment:
It's better to make these into hooks so when you update the value it call the Function on the client with the new value, Without the networking having to check constantly If it has changed, 
https://unitystation.github.io/unitystation/development/SyncVar-Best-Practices-for-Easy-Networking/

Reply:
as I mentioned above, these shouldn't really be syncvars in the first place